src/components/FilterBadges/index.jsx

```diff
@@ -0,0 +1,150 @@
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import initialFilters from '../../data/constants/filters';
+
+/*
+  Active filter values are stored in GradebookPage State, passed into props in selected* variables
```

Comment:
I typed this out for myself as I was working to think through some of it, and I was gonna delete it, but .... maybe it would just be better to leave it here?

Reply:
You should definitely delete this. A subcomponent shouldn't care about how Gradebook and GradebookPage are written, which may well change without this component changing.